csrankings-2.csv

```diff
@@ -1095,6 +1096,7 @@ Eric Dinet,Université Jean Monnet,https://laboratoirehubertcurien.univ-st-etien
 Eric Dubois 0002,University of Ottawa,http://www.site.uottawa.ca/~edubois,1KEbwBkAAAAJ
 Eric E. Aubanel,University of New Brunswick,http://www.cs.unb.ca/~aubanel,5KrloWkAAAAJ
 Eric E. Gilbert,University of Michigan,http://eegilbert.org,71OktocAAAAJ
+Eric Feron,KAUST,https://www.kaust.edu.sa/en/study/faculty/eric-feron,wKvaIJgAAAAJ

```

Comment:
Not in CS; doesn't clearly meet the inclusion criteria.

Reply:
OK, I have deleted it.
```suggestion

```